eng/pipelines/azure-pipelines.yml

```diff
@@ -133,7 +133,7 @@ stages:
       - ${{ if eq(variables._RunAsPublic, True) }}:
         - job: linux
           # timeout accounts for wait times for helix agents up to 30mins
-          timeoutInMinutes: 60
+          timeoutInMinutes: 90
```

Comment:
IIUC, we are enabling signing only on the internal pipeline. Do we need to bump this timeout then?

Reply:
hrm... good point, I can't remember from the top of my head but I do remember we had to do this in other previews too for some reason. Given this is mostly just an increment that would not matter much and we won't be doing this in main, I'd probably rather keep it increased just in case.